rotkehlchen/tests/unit/decoders/test_main.py

```diff
@@ -131,3 +134,97 @@ def test_no_logs_and_zero_eth(
+    from_address = ethereum_accounts[0]
+    to_address = ethereum_accounts[1]
+    tether_address = string_to_evm_address('0xdAC17F958D2ee523a2206206994597C13D831ec7')
+    transaction = EvmTransaction(
+        tx_hash=evmhash,
+        timestamp=0,
+        block_number=0,
+        from_address=from_address,
+        to_address=tether_address,
+        value=0,
+        gas=0,
+        gas_price=0,
+        gas_used=0,
+        input_data=b'',
+        nonce=0,
+    )
+    receipt = EthereumTxReceipt(
+        tx_hash=evmhash,
+        contract_address=None,
+        status=True,
+        type=0,
+        logs=[
+            EthereumTxReceiptLog(
+                log_index=73,
+                data=hexstring_to_bytes('0x000000000000000000000000000000000000000000000000000000000243de35'),  # noqa: E501
+                address=tether_address,
+                removed=False,
+                topics=[
+                    hexstring_to_bytes('0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef'),  # noqa: E501
+                    hexstring_to_bytes('0x0000000000000000000000004bba290826c253bd854121346c370a9886d1bc26'),  # noqa: E501
+                    hexstring_to_bytes('0x000000000000000000000000ed2f12b896d0c7bff4050d3d8c4f95bd61aaa12d'),  # noqa: E501
+                ],
+            ),
+        ],
+    )
+    dbethtx = DBEthTx(database)
+    with database.user_write() as cursor:
+        dbethtx.add_ethereum_transactions(cursor, [transaction], relevant_address=None)
+        events = evm_transaction_decoder.decode_transaction(
+            write_cursor=cursor,
+            transaction=transaction,
+            tx_receipt=receipt,
+        )
+    assert events == [
+        HistoryBaseEntry(
+            event_identifier=HexBytes(
+                '0xbb58b36ddc027a1070131e68b915e5f0dca37767b020ed164eda681725b5ca4e',
+            ),
+            sequence_index=0,
+            timestamp=0,
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(),
+            location_label='0x4bBa290826C253BD854121346c370a9886d1bC26',
+            notes='Burned 0 ETH for gas',
+            counterparty='gas',
+            identifier=None,
+            extra_data=None,
+        ), HistoryBaseEntry(
+            event_identifier=HexBytes(
+                '0xbb58b36ddc027a1070131e68b915e5f0dca37767b020ed164eda681725b5ca4e',
+            ),
+            sequence_index=74,
+            timestamp=0,
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.TRANSFER,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_USDT,
+            balance=Balance(amount=FVal('38.002229')),
+            location_label=from_address,
+            notes=f'Send 38.002229 USDT from {from_address} to {to_address}',
```

Comment:
Actually can you also change the verb to `Transfer` in decoding/base.py? It may make sense to use a different verb too for the TRANSFER type